components/Navigation.js

```diff
@@ -0,0 +1,17 @@
+import Link from "next/link";
+
+export default function Navigation() {
+  return (
+    <ul>
+      <li>
+        <Link href={"/"}>Spotlight</Link>
+      </li>
+      <li>
+        <Link href={"/art-pieces"}>Art Pieces</Link>
+      </li>
+      <li>
+        <Link href={"/favorites"}>Favorites</Link>
+      </li>
+    </ul>
```

Comment:
I recommend wrapping the unordered list in a nav-tag so that you have a semantic navbar

and you don't need the {} wrapping the links, since they are strings